keepassxc-browser/background/httpauth.js

```diff
@@ -73,16 +73,37 @@ httpAuth.processPendingCallbacks = async function(details, resolve, reject) {
 };
 
 httpAuth.loginOrShowCredentials = function(logins, details, resolve, reject) {
-    // At least one login found --> use first to login
+    // At least one login found
     if (logins.length > 0 && page.settings.autoFillAndSend) {
-        if (logins.length === 1) {
+        let loginToUse = null
+        if(logins.length === 1){
```

Comment:
Please don't do this. Just because there is only a single entry iny my KeepassXC database for a given site, this doesn't mean that I'm fine with the browser extension sending those credentials automatically to the server without asking me first.

Please *only* send credentials for which the HttpAuth flag has been set, regardless of how many entries there are for the same site.